### PR TITLE
Login: Update the login screen to match the mockups

### PR DESCRIPTION
**Pull request by scruffian**
This pull request updates the design of the login form based on the designs in p6qnuF-4lz-p2.
  
The designs don't match the mockups exactly, as I have tried to stick to existing Calypso colours where possible. @josemarques, can you review these difference and see what you think?

#### Testing instructions
  
1. Run `git checkout update/login-design` and start your server, or open a [live branch](https://delphin.live/?branch=update/login-design)
2. Open the [`Login` page](http://calypso.localhost:3000/login)
3. Check that the design matches the mockup
  
#### Additional notes

There is more work to do with notices, but this PR is big enough already.
  
#### Reviews
  
- [x] Code
- [x] Product
- [ ] Design   

@Automattic/sdev-feed

### CODE REVIEW

File: client/login/wp-login/index.js [hunk 1]
```diff
@@ -77,22 +94,13 @@ class Login extends React.Component {
...
-							<div>{
-								// @TODO show currently logged in user if any
-								translate( 'You are signed out' )
-							}</div>
-						</div>
 						<div className="wp-login__container">
 							{ showingMagicLoginRequestForm
 								? <RequestLoginEmailForm />
-								: <LoginBlock title={ translate( 'Sign in to WordPress.com' ) } />
+								: <LoginBlock title={ translate( 'Log in to your account.' ) } />
```
Comment (a8ci18n) [inline client/login/wp-login/index.js]: Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Log in to your WordPress.com account' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by gziolo** [comment]
@jblz or @scruffian I tried to rebase this PR, but it requires more knowledge than I expected. Could you do it?

File: client/login/wp-login/index.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ import {
 	isShowingCheckYourEmailPage,
 } from 'state/login/magic-login/selectors';
 import { getCurrentQueryArguments } from 'state/ui/selectors';
-
+import Gridicon from 'gridicons';
```
Comment (gziolo) [inline client/login/wp-login/index.js]: This is external dependency.

File: client/login/wp-login/index.js [hunk 3]
```diff
@@ -50,21 +50,38 @@ class Login extends React.Component {
 		}
 	}
 
+	goBack( event ) {
+		event.preventDefault();
+
+		if ( typeof window !== 'undefined' ) {
+			window.history.back();
```
Comment (gziolo) [inline client/login/wp-login/index.js]: In some places we use also `page.back()`, but I'm not sure if that makes any difference.

**Comment by gziolo** [comment]
It looks like it needs some adjustments from @josemarques. Code changes made so far look good.

**Comment by scruffian** [comment]
@gziolo, If you're happy with the code changes, it might be worth merging this and letting @josemarques do another review later, to prevent conflicts down the road...

**Comment by scruffian** [comment]
rebase ✅

File: client/login/wp-login/index.js [hunk 4]
```diff
@@ -93,22 +110,13 @@ class Login extends React.Component {
...
-							<div>{
-								// @TODO show currently logged in user if any
-								translate( 'You are signed out' )
-							}</div>
-						</div>
 						<div className="wp-login__container">
 							{ magicLoginView === REQUEST_FORM
 								? <RequestLoginEmailForm />
-								: <LoginBlock title={ translate( 'Sign in to WordPress.com' ) } />
+								: <LoginBlock title={ translate( 'Log in to your account.' ) } />
```
Comment (a8ci18n) [inline client/login/wp-login/index.js]: Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Log in to your WordPress.com account' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (gziolo) [inline client/login/wp-login/index.js]: Updating to match bot's suggestion.

**Comment by gziolo** [comment]
Let's merge it as it is. We can iterate in another PR. :shipit:

**Pull request merged**